config/default/ui-patch-init-containers.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocp-metal-ui
+  namespace: assisted-installer
+spec:
+  template:
+    metadata:
+      labels:
+        app: ocp-metal-ui
+    spec:
+      initContainers:
+      - name: init-wait-for-route
+        image: quay.io/openshift/origin-cli:latest
+        command: ['sh', '-c', "until oc get routes -n $(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace) assisted-service; do echo waiting for assisted-service route; sleep 2; done"]
+      - name: init-update-proxy-pass-in-ngnix-conf-config-map
+        image: quay.io/openshift/origin-cli:latest
+        command: ['sh', '-c', 'export NAMESPACE=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace) && export ROUTE_URL=$(oc get routes -n $NAMESPACE assisted-service -o jsonpath={.spec.host}) && if [ "$ROUTE_URL" != "" ] ; then oc get configmap -n $NAMESPACE ocp-metal-ui -o yaml | sed -e "s|assisted-service.assisted-installer.svc.cluster.local|$ROUTE_URL|" | oc apply -f - ; fi']
```

Comment:
Please change the command field to have [multi-line](https://yaml-multiline.info/) content

Reply:
@YuviGold apart from this, does the rest look good?

Reply:
I would not invest extra effort into it. Anyhow it will be removed soon. If it works and does not break anything, ship it